src/components/core/DpEditor/DpEditor.vue

```diff
@@ -1153,6 +1153,9 @@ export default {
           // Strip img tags from pasted and dropped content
           returnContent = returnContent.replace(/<img.*?>/g, '')
 
+          // Strip line break spaces, white spaces and new lines
+          returnContent = returnContent.replace(/(&nbsp;|\r|\n)/gmi, '')
```

Comment:
This looks like all those characters are replaced by the empty string, which leads to all spaces between words being dropped. I guess this is not what we want. I'd suggest to replace the characters with one space character.

```suggestion
          returnContent = returnContent.replace(/(&nbsp;|\r|\n)/gmi, ' ')
```

Reply:
thank you, it thought i put a space in already

Reply:
done https://github.com/demos-europe/demosplan-ui/pull/80/commits/82f72fb092309951b759dd14d6ad171507ea40e1